src/content/learn/preserving-and-resetting-state.md

```diff
@@ -494,7 +494,7 @@ label {
 
 You might expect the state to reset when you tick checkbox, but it doesn't! This is because **both of these `<Counter />` tags are rendered at the same position.** React doesn't know where you place the conditions in your function. All it "sees" is the tree you return.
 
-In both cases, the `App` component returns a `<div>` with `<Counter />` as a first child. To React, these two counters have the same "address": the first child of the first child of the root. This is how React matches them up between the previous and next renders, regardless of how you structure your logic.
+In both cases, the `App` component returns a `<div>` with `<Counter />` as a first child. To React, these two counters have the same "address": the first child of the root. This is how React matches them up between the previous and next renders, regardless of how you structure your logic.
```

Comment:
These wordings seems correct? 

`<Counter />` is the first child of the first child (div) of the root?

Reply:
Yes @harish-sethuraman you are right, it is correct. Sorry I missed the detail.
Thanks.